package/kernel/linux/modules/netsupport.mk

```diff
@@ -922,6 +922,22 @@ endef
 $(eval $(call KernelPackage,sched-ipset))
 
 
+define KernelPackage/sched-fq-pie
+  SUBMENU:=$(NETWORK_SUPPORT_MENU)
+  TITLE:=Flow Queue w Proportional Integral Enhanced (FQ-PIE)
+  DEPENDS:=+kmod-sched-core +kmod-sched
```

Comment:
Do you really have to depend on `+kmod-sched`?
Please check if `+kmod-sched-core` is sufficient.

Reply:
Was the only way I found to quiesce a dependency error generated on PIE

Reply:
What dependency error do you get without the `+kmod-sched` dependency?

Reply:
Well, hmm... Dried muffin remnant I guess, updated and cleaned buildroot, no error

Reply:
With good reason, it's no longer building the kmod. If I make kmod-sched = m I get the error first encountered:
`Package kmod-sched-fq-pie is missing dependencies for the following libraries:
sch_pie.ko
`
not sure what is forcing this.

Reply:
Decided to break PIE out of the big kmod-sched

Reply:
@anomeome I wonder if splitting PIE from sched would produce some side effects. I assume sqm packages assume PIE is always present so making it an optional package can be problematic?

Reply:
Ya, I had originally rejected this idea due to concern of unknown dependency issues, but taking a second look at a grep on _kmod-sched_ of the _feeds/_ and _package/_ hierarchies leads me to think it will be OK. But, if a concern, it can certainly be reverted to the original.

Reply:
@anomeome my main concern is with sqm scripts that for sure needs to be updated... Can't remember if the pie script are installed by default or they are in -extras package. Can you check that? Once we have checked that I can merge this.

Reply:
Nothing in the sqm-scripts Makefile dependency list forces _kmod-sched_, which would be the only way to force PIE.

Reply:
Ok nice, sorry for the mess...